src/Smart.Design.Razor/TagHelpers/Constants/Color.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace Smart.Design.Razor.TagHelpers.Constants
+{
+    public static class Color
+    {
+        public const string BackgroundLight = "#F6F6F6";
```

Comment:
I'm wondering if we shouldn't try to link back to the design system web site more often.

Reply:
Yep it aint mentioned neither in the commit message.
Also, I have a script that generates the Class based on https://github.com/smartcoop/design/blob/a17a0a4e32c3d48a2a89ab2cf0e41812658b8b9d/content/scss/_s-colors.scss.
It is not included in the repo and I wonder, if shared, where it should go (public or Smart internally)

Reply:
I guess we can include the script in a `scripts/` directory and mention it at the top of the file here, and mention within the script that it is used to generate this file (I do a lot of crossreferencing comments in cases like these, so that people who discover the code base in any order can follow easily what's going on).